modules/ROOT/pages/7.4-release-notes.adoc

```diff
@@ -160,6 +160,15 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== The image dialog lost focus after closing an image upload error alert.
+// #TINY-11159
+
+Previously, when an image upload error alert was closed, the image dialog lost focus, impacting keyboard accessibility.
+
+{productname} {release-version} addresses this issue by making the dropzone component focusable and adding a callback to focus the dropzone button after the error alert is closed.
+
+As a result, keyboard users now experience improved accessibility, with the ability to focus on the dropzone using the dialog API.
```

Comment:
There was no change in the dialog API, the `focus` was already there. It is just that the dropzone is now focusable so it can be used to focus the dropzone component.